### ISSUE TITLE
PEx and PWC+

### ISSUE THREAD

**Issue by pinoylifecraft** [NovaFox161/PerWorldChatPlus#2]
Hello there It's me again I have a Probsss about PermissionEx and PWC+
I can't see our Prefix in chat box
![16507221_1386469564707707_371768337_n](https://cloud.githubusercontent.com/assets/13610530/22624312/fb5369ca-ebb2-11e6-93dc-20371547a4cf.png)
This is our Prefix instead " GM : " So sad :disappointed:

**Comment by NovaFox161** [comment]
Yes, the chat formatting is in beta in the most recent release of PWC+
In some of the latest commits this should be fixed as there is a full system for chat formatting now.
Currently the latest version of PWC+ is now in testing.

**Comment by pinoylifecraft** [comment]
I hope this should work fine in the next update and all my BUGS Report will fix and + support with no error in 1.11.2 soon as I'll Update my server soon :+1:
